Source/SPDatabaseDocument.m

```diff
@@ -512,6 +512,18 @@ - (void)setConnection:(SPMySQLConnection *)theConnection
 	}
 #endif
 
+	#ifndef SP_CODA /* set connection time zone from prefs */
+	// Set the connection time zone if necessary
+	NSString *timeZoneIdentifier = [prefs objectForKey:SPDefaultTimeZone];
+	if ([timeZoneIdentifier isEqualToString:@"Use Server Time Zone"]) {
+		// do nothing
+	} else if ([timeZoneIdentifier isEqualToString:@"Use System Time Zone"]) {
```

Comment:
I don't really like the strings... Perhaps we have some special key for each of these two things (see SPConstants)

Reply:
Yes, I will likely split it into two defaults. An int that denotes Server/Local/Fixed TZ, and a string for the selected TZ.